### ISSUE TITLE
Question/Feature Request: Release 'easytable' as JPMS module

### ISSUE THREAD

**Issue by cnr-roxx** [vandeseer/easytable#170]
Hi,

First of all: Big thank you for this very usefull piece of code!

Questions: 

Do you have plans to release 'easytable' as JPMS module?

If not: Would you be so kind and point me to info or resources - how can I use your library in my Java22 JPMS app?

Thank you once again and regards!

**Comment by vandeseer** [comment]
Fair point. Currently I don't use the library not actively anymore. However I try to support it still as my contribution to the Open Source community...

So no plans currently, but if you come up with a pull request I am happy to merge;)
